event-processing/lambda/models/cleansed-event.ts

```diff
@@ -12,18 +21,23 @@ export class CleansedEvent implements ICleansedEvent {
     readonly component_id: string;
     readonly timestamp: number;
     readonly timestamp_formatted: string;
+    readonly extensions?: unknown | undefined;
 
     constructor(
         event_id: string,
         event_name: string,
         component_id: string,
         timestamp: number,
         timestamp_formatted: string,
+        extensions?: unknown | undefined,
     ) {
         this.event_id = event_id;
         this.event_name = event_name;
         this.component_id = component_id;
         this.timestamp = timestamp;
         this.timestamp_formatted = timestamp_formatted;
+        if (extensions != undefined && (extensions as ICleansedExtensionsEvent).evidence) {
+          this.extensions = { evidence : { validityScore : ((extensions as ICleansedExtensionsEvent).evidence as ICleansedEvidenceEvent).validityScore } };
```

Comment:
if the validityScore does not exist, this will cause it to be assigned undefined. So when we JSON.stringify in [cleanser-app.ts#L31](https://github.com/alphagov/di-txma-audit/blob/8ff4ebb5fc09f0c82cfd12d06a40917229682cc5/event-processing/lambda/cleanser-app.ts#L31), you are left with `"extensions": {"evidence" : {}}`. Is this desired behaviour?

Reply:
That's not a bad thing, but we might want to remove it. Creating a new PR now